permissions/plugin-thycotic-vault.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "thycotic-vault"
+github: "jenkinsci/thycotic-vault-plugin"
+paths:
+  - "com.thycotic/plugins/thycotic-vault"
```

Comment:
You need to replace all `.` with `/`. Also, it looks like your `groupId` is just `com.thycotic` and your `artifactId` is `thycotic-valut`, so your path would be `com/thycotic/thycotic-vault`

Reply:
Fixed!!